NonWeb.md

```diff
@@ -20,8 +21,19 @@ both browsers and node.js. This situation is expected to be first encountered
 with POSIX features such as file I/O. In that respect, WebAssembly would err
 towards standardizing existing practice through libraries, and let developers
 choose which libraries to use.
+Similarly, the WebGL and SDL interface (
+[exposed by emscripten](http://kripken.github.io/emscripten-site/docs/porting/multimedia_and_graphics/OpenGL-support.html)
+in C/C++) could be made into a shared spec with non-browser mobile/desktop
+environments, allowing for cross platform browser & non-browser games and other
+things.
```

Comment:
For SDL, should there be an SDL-level spec, or lower-level input/timing/audio specs with SDL being just a library on top of them? I personally don't have a sense of what's best here. But if you want portability to browsers, we have Web APIs with specs for the low-level things today.